src/compat/posix/time/strptime.c

```diff
@@ -0,0 +1,327 @@
+/**
+ * @file strptime.c
+ * @brief writing linux function strptime for embox os
+ *
+ * author Tamara Deryugina <tamara.deryugina@gmail.com>
+ * @date 02.03.2020
+ */
+
+#define _XOPEN_SOURCE
+#include<stdio.h>
+#include<stdlib.h>
+#include<string.h>
+#include<time.h>
+#include<ctype.h>
+#include<math.h>
+
+int str_legth(const char *s) {
+	int count = 0;
+	while(s[count] != '\0') {
+		count++;
+	}
+	return count;
+ }
+
+int num_length(char *new_s) {
+	int count = 0;
+	while ((new_s[count] > 47) && (new_s[count] < 58)) {
+		count++;
+	}
+	return count;
+}
+
+int word_length(const char *s) {
+	int count = 0;
+	while ((s[count] > 96) && (s[count] < 123)) {
+		count++;
+	}
+	return count;
+}
+
+int degree(int dec, int num) {
+	int count = 0;
+	int res = 1;
+	if (num == 0) {
+		return 1;
+	}
```

Comment:
No new line between '}' and `else` is required according to the code style

Reply:
I fixed it, thank you!